### ISSUE TITLE
BetterFountain stops the Vim extension working

### ISSUE THREAD

**Issue by lkingsford** [piersdeseilligny/betterfountain#51]
I'm not sure about the reason, but when I am using Better Fountain, the Vim extension stops working.

Specifically, it stops the Vim hotkeys working if I load a .fountain file when BetterFountain is installed. If I disable and then reenable BetterFountain, then the Vim keys will continue to work.

Love the extension, but this is making it a little difficult for my vim addicted fingers to work. My suspicion is that it may be related to the autocorrect - but that's the vaguest hunch. I'll do some more investigation myself.

**Comment by piersdeseilligny** [comment]
@lkingsford I can't seem to reproduce this; does it still happen with the 1.4.5 version of the extension?

**Comment by lkingsford** [comment]
Tested on Mac - seems to be fixed with latest extension (or some time in the last month or two). Thank you. Will verify on the Windows machine soon.

**Comment by stoneG** [comment]
When I load up VSCode with Vim (enabled globally) and Better Fountain (enabled for workspace), I'm seeing that Vim hotkeys don't work (ie. esc, h, j, k, l).

I need to disable Better Fountain, reload VSCode, then re-enable Better Fountain for Vim hotkeys to work.
```
Version: 1.47.2 (user setup)
Commit: 17299e413d5590b14ab0340ea477cdd86ff13daf
Date: 2020-07-15T18:22:06.216Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362

Vim 1.15.0
Better Fountain 1.6.5
```

**Comment by luisfontes** [comment]
Stumbled on this as well. I suspect is related with the autocomplete. When the Vim extension is not working the Fountain autocomplete works, when the Vim extension is working the autocomplete does not.

The steps I have done to successfully reproduce is to have a .fountain file open as the active editor and then reload VSCode. When VSCode reloads and the active editor is a fountain file the vim keybindings stop working and the fountain autocomplete works, when VSCode reloads and the active editor is not a fountain file the inverse happens.

With the devtools opened these are the errors that pop up:

After reload and active editor is a fountain file
<img width="546" alt="image" src="https://user-images.githubusercontent.com/5988029/98469176-cf466f00-21bc-11eb-8410-66092743a6cb.png">



After reload, active editor not a fountain file then I open the fountain file
<img width="542" alt="image" src="https://user-images.githubusercontent.com/5988029/98469237-159bce00-21bd-11eb-8a7f-0815b00ba917.png">

**Comment by luisfontes** [comment]
Maybe related: #84

**Comment by piersdeseilligny** [comment]
@luisfontes yep #84 is a duplicate of this one, although I accidentally closed this issue despite it not being fixed - so i guess #84 is the active issue for this.

For the moment, I'm going to attempt to simply show a warning if the `type` command fails to register (so that it at least doesn't completely fail to load the extension) as i'm not actually sure if this can be fixed, without vscode allowing multiple extensions to register it.
